mode.c

```diff
@@ -224,6 +224,7 @@ void init_vm_mode(void) {
 	behavior_start(B_LEDS_PROX);
 	behavior_start(B_SOUND_BUTTON);
 	behavior_start(B_LEDS_MIC);
+	prox_disable_network();
```

Comment:
you should also do this:
 			CLEAR_EVENT(EVENT_DATA);
 			vmVariables.rx_data = 0;
 			vmVariables.rx_data = 0;
 			vmVariables.ir_tx_data = 0;

(have a look in MODE_FOLLOW exit).